files/update_docker_image.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Pulls a docker image.
+# Returns 0 if there a change.
+# Returns 1 if there is no change.
+# Returns 3 if something when wrong.
+#
+DOCKER_IMAGE=$1
+
+BEFORE=$(docker inspect --format='{{.Config.Image}}' ${DOCKER_IMAGE} 2>/dev/null)
```

Comment:
The previous implementation allowed overriding the binary with `docker_command` but this script doesn't do that.